packages/sdk/src/evm/core/classes/transactions.ts

```diff
@@ -1043,7 +1043,7 @@ async function enginePrepareRequest(
 
       const domain = {
         name,
-        version: "1",
+        version: gaslessOptions.engine.domainSeparatorVersion,
```

Comment:
Since the `domain` details is used in `signTypedDataInternal`, we also need to make this `dynamic`, right. as this is currently hard-coded to `1`. @joaquim-verges

Reply:
this is outdated